src/bundler/bundle_v2.zig

```diff
@@ -3177,8 +3447,173 @@ pub const ParseTask = struct {
         }
     }
 
-    pub fn onComplete(result: *Result) void {
-        BundleV2.onParseTaskComplete(result, result.ctx);
+    pub fn onComplete(result: *Result) void {
+        BundleV2.onParseTaskComplete(result, result.ctx);
+    }
+};
+
+/// Files for Server Components are generated using `AstBuilder`. This is
+/// equivalent to a ParseTask, including using the same completion callback, but
+/// it does not take a source code string.
+pub const GenerateTask = struct {
```

Comment:
```suggestion
pub const ServerComponentsParseTask = struct {
```

Reply:
`Parse` is a bad name to include since it does not invoke js_parser. since it produces ParseResult, that is OK